### ISSUE TITLE
WSL (potential bug) with DBUS and vnc

### ISSUE THREAD

**Issue by RishRishTheFish** [microsoft/WSL#10168]
Im using wsl2 btw, ubuntu mate

Following this tutorial: https://ywjheart.wordpress.com/2019/01/09/install-mate-desktop-on-wsluse-linux-desktop-on-windows-10/
[message_7.txt](https://github.com/microsoft/WSL/files/11673653/message_7.txt)
^ so im having a issue with getting the mate-session to work, this is a issue with wsl itself since it does not always create a bus folder in the directory where it listens on BUS.

My dbus-1 configuration:
< busconfig >
  
  <listen>unix:path=/run/user/0/bus</listen>

  
< policy group="power" >
  < allow send_destination="org.freedesktop.PowerManagement"/ >
  < allow send_interface="org.freedesktop.PowerManagement"/ >
< /policy >

  

< /busconfig >

Tldr: Im trying to set up a desktop GUI (see the Ubuntu distro in mate) on a separate application, im trying to use xlserver (i dont expect any assistance in that regard) or ill try and use ssh. basically, i can see the apps and use it like a regular linux distro. 
I don't think i put all the needed information so if you'd like any info like logs im happy to oblige. (when i run dpkg-reconfigure dbus && service dbus restart and restart it, the results are slightly promising but ultimately fail, regardless the vnc is glitchy and id like it to work normally)

**Comment by OneBlue** [comment]
WSL 1.3.10 fixed an issue with  /run/user not being writable by users, does upgrading to [1.3.10](https://github.com/microsoft/WSL/releases/tag/1.3.10)  solve the issue ?
